src/components/ChatLog.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+
+const ChatLog = (props) => {
+    const entireChatLog = props.entries.map((chatEntry) => {
+        return (
+            <section>
+            <ChatEntry
+            id={chatEntry.id}
+            sender={chatEntry.sender}
+            body={chatEntry.body}
+            timeStamp={chatEntry.timeStamp}
+            liked={chatEntry.liked}
+            onUpdate={props.onUpdateChat}
+            >
+            </ChatEntry> </section>
```

Comment:
While I think the use of a `section` tag here is good, it does mean that there are no actual `li` tags in the `ul`, which is a minor violation of HTML conventions.